run_dir/design/flowcells.html

```diff
@@ -208,17 +214,12 @@ <h1><span id="page_title">Flowcells</span> </h1>
         } );
     } );
 }
-$(".running-note-panel").each(function(i){
-  var data=$(this).text().split('+=+');
-  notedate = new Date($.trim(data[0]));
-  run_note=JSON.parse(data[1]);
-  $(this).replaceWith(
-    "<div class='panel-heading'> \
-    <a href='mailto:"+run_note['email']+"'>"+run_note['user']+"</a> - "+ notedate.toDateString()+", " + notedate.toLocaleTimeString(notedate)+" \
-    </div> \
-    <div class='panel-body' style='white-space: pre-line'> "+make_markdown(run_note['note'])+ "\
-    </pre> \
-    </div>");
+
+//Formatting for Running note panel body
+$(".running-note-panel > .panel-body").each(function(i){
+  var data=$(this).text();
+  run_note=JSON.parse(data);
```

Comment:
Why are we printing JSON in the HTML? Why not just print the `note` key directly and then `make_markdown` on the text?